src/els_config.erl

```diff
@@ -99,6 +100,7 @@ do_initialize(RootUri, Capabilities, {ConfigPath, Config}) ->
   CodeReload = maps:get("code_reload", Config, disabled),
   Runtime = maps:get("runtime", Config, #{}),
   CtRunTest = maps:get("ct-run-test", Config, #{}),
+  ElvisPath = maps:get("elvis_path", Config, undefined),
```

Comment:
This maybe interpreted as the path to the `elvis` executable/escript. Would `elvis_config_path` be a more precise name?

Reply:
Good point, fixed.